lib/rucio/daemons/hermes/kafka/filters/filtered_list.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from rucio.daemons.hermes.kafka.filters.default import default
+from rucio.client import Client
+
+class filtered_list(default):
+
+    RUBIN_BUTLER = "rubin_butler"
```

Comment:
So this file is the really Rubin-specific component.  It wouldn't be part of the Rucio distribution, but we'd add it on for Rubin deployments, correct?

Reply:
yes